server/game/cards/09.4-TCoH/UnspokenEttiquette.js

```diff
@@ -0,0 +1,20 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class UnspokenEtiquette extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Dishonor each participating non-courtier',
+            effect: 'dishonor each participating non-courtier.',
+            condition: context => context.game.currentConflict && context.game.currentConflict.conflictType === 'political',
```

Comment:
As mentioned previous `context.game.isDuringConflict('political')` is much more concise.